receiver/k8sclusterreceiver/e2e_test.go

```diff
@@ -68,47 +70,89 @@ func TestE2E(t *testing.T) {
 
 	wantEntries := 10 // Minimal number of metrics to wait for.
 	waitForData(t, wantEntries, metricsConsumer)
-	// golden.WriteMetrics(t, expectedFile, metricsConsumer.AllMetrics()[len(metricsConsumer.AllMetrics())-1])
```

Comment:
nit: I would suggest we don't remove this since it's useful to just uncomment that line in order to update the golden files. Maybe adding a comment to describe its usage would also help. Same applies for the newly added test.

Reply:
good catch, thanks - I re-added the comment